vagrant/pg_config.sh

```diff
@@ -1,7 +1,8 @@
 apt-get -qqy update
 apt-get -qqy install postgresql python-psycopg2
 apt-get -qqy install python-flask python-sqlalchemy
-apt-get -qqy install python-pip
+wget https://bootstrap.pypa.io/get-pip.py
+python ./get-pip.py
```

Comment:
I've gone into this issue some more and it seems to be the act of installing `bleach` that breaks `pip`. The `bleach` package does not install its dependencies in the correct order. This can be solved by installing them before `bleach` within our script:

```
pip install packaging
pip install appdirs
pip install -U six
```

I've tested this solution and it works a treat. Much better than my first proposal. Thanks for making me think some more about this issue. I will update the pull request with this solution.